<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>M17ChannelDialog</class>
 <widget class="QDialog" name="M17ChannelDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="basicTab">
      <attribute name="title">
       <string>Basic</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="nameLabel">
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="channelName">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="rxFrequencyLabel">
           <property name="text">
            <string>Rx Frequency</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="rxFrequency"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="txFrequencyLabel">
           <property name="text">
            <string>Tx Frequency</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="txFrequency"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="powerLabel">
           <property name="text">
            <string>Power</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="txTimeoutLabel">
           <property name="text">
            <string>Tx Timeout</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="rxOnlyLabel">
           <property name="text">
            <string>Rx Only</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QCheckBox" name="rxOnly"/>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="scanListLabel">
           <property name="text">
            <string>Scan List</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QComboBox" name="scanList"/>
         </item>
         <item row="3" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QComboBox" name="powerValue">
             <item>
              <property name="text">
               <string>Max</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>High</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Mid</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Low</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Min</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="powerDefault">
             <property name="text">
              <string>Default</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="4" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QSpinBox" name="totValue">
             <property name="toolTip">
              <string>Specifies the transmit timeout in seconds. A value of 0 means that the transmit timeout is disabled.</string>
             </property>
             <property name="specialValueText">
              <string>Off</string>
             </property>
             <property name="suffix">
              <string> s</string>
             </property>
             <property name="maximum">
              <number>9999</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="totDefault">
             <property name="text">
              <string>Default</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="voxLevelLabel">
           <property name="text">
            <string>VOX Level</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QSpinBox" name="voxValue">
             <property name="specialValueText">
              <string>Off</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="voxDefault">
             <property name="text">
              <string>Default</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Channel Mode</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="channelMode">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="currentIndex">
            <number>2</number>
           </property>
           <item>
            <property name="text">
             <string>Voice</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Data</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Voice + Data</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="colorCodeLabel">
           <property name="text">
            <string>Access Number</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="accessNumber">
           <property name="maximum">
            <number>15</number>
           </property>
           <property name="value">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="txContactLabel">
           <property name="text">
            <string>Tx Contact</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="txContact"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="posSysLabel">
           <property name="text">
            <string>Send position</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QCheckBox" name="sendGPS">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="extTab">
      <attribute name="title">
       <string>Extended</string>
      </attribute>
      <attribute name="toolTip">
       <string>Device specific settings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="ExtensionView" name="extensionView" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ExtensionView</class>
   <extends>QWidget</extends>
   <header>extensionview.hh</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>M17ChannelDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>M17ChannelDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
